test/helper/formatCertificate.js

```diff
@@ -0,0 +1,41 @@
+module.exports = function formatCertificate (txJson) {
+    var txObj = {};
+    txObj.version = txJson.version;
+
+    const  { scid, ...cert } = txJson.cert
+
+    txObj.cert = {
+        ...cert, ...{
+            sidechainId: txJson.cert.scid,
+            ftScFee: txJson.cert.ftScFee * 100000000,
+            mbtrScFee: txJson.cert.mbtrScFee * 100000000
+        }
+    }
+
+    let formattedInput = [];
+    txJson.vin.forEach(input => {
+        formattedInput.push({
+            output: { hash: input.txid, vout: input.vout }, 
+            script: input.scriptSig.hex, 
+            sequence: "ffffffff", // input.sequence?
```

Comment:
Should the `sequence` match?

Reply:
should be ok a fixed value according to @MarcoOl94